src/apb/entrypoint.py

```diff
@@ -50,12 +59,15 @@ def get_last_run(
             f"No previous runs for {workflow_id} in {repo.full_name}, {response.status_code}"
         )
         return None
-    workflow_runs = response.json()["workflow_runs"]
-    if len(workflow_runs) == 0:
-        return None
-    else:
-        last_run_date = workflow_runs[0]["created_at"]
-        return isoparse(last_run_date).replace(tzinfo=None)
+
+    # Find the date of the most recent workflow run against the default branch.
```

Comment:
Nitpick but we typically omit periods for one-line comments.

```suggestion
    # Find the date of the most recent workflow run against the default branch
```